Feature/logger

ibcylon opened the pull request:
## Motivation ⍰

- Logger 생성
- info(),debug(), error(), notice() 등 사용할 수 있음

<br>

## Key Changes 🔑

- xcode 15 사용 시 편의성 및 가독성 좋음, 아이콘, 색상, 카테고리별로 그루핑해서 볼 수도 있음
- console 앱, 프로파일링 앱에서 통합된 로그를 볼 수 있음, 로그에 중요한 정보는 암호화 할 수 있음.

<br>

## To Reviewers 🙏🏻

`TFLogger.viewCycle.debug("asdf")`

<br>

## Linked Issue 🔗

- #29

ibcylon committed:
Add Logger

ibcylon committed:
Fix: prefix Logger name

Minny27 commented on Falling/Sources/Util/Logger+Util.swift:
```diff
@@ -0,0 +1,27 @@
+struct TFLogger {
+  static var viewCycle = Logger.viewCycle
+
+  static let data = Logger.data
```
viewCycle과 data의 차이를 알 수 있을까요?

Minny27 committed:
[#27]Refactor: 로그 카테코리를 view, logic, data로 나눔

- MVVM 패턴을 사용하기 때문에 이렇게 나누자고 의견이 모아짐
- view는 presentation logic
- logic은 business logic
- data는 서버로부터 응답받은 데이터 + storage 데이터

Minny27 commented:
MVVM 아키텍처 패턴을 사용하기 때문에 로그 카테코리를 view, logic, data로 나누게 됨
- view는 presentation logic
- logic은 business logic
- data는 서버로부터 응답받은 데이터 + storage 데이터
